resources/core/charts/service-catalog/charts/catalog-ui/templates/virtualservice.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.virtualservice.enabled }}
+{{- $serviceName := include "fullname" . -}}
+{{- $servicePort := .Values.service.externalPort -}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+{{- if .Values.virtualservice.annotations }}
+  annotations:
+{{ toYaml .Values.virtualservice.annotations | indent 4 }}
+{{- end }}
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "alertmanager.name" . }}
```

Comment:
are you sure "alertmanager.name"

Reply:
You're right ;) fixed in more places as well